### DISCUSSION TITLE
Release: 'jnode-revisited' website on GitHub Pages

### DISCUSSION THREAD

**Discussion by ghost** [jnode-revisited/jnode#6] [Q&A] [answered]
Hi everyone, hi @tripleo1.

#### feedback

> Nice start. Send a PR when you're ready.

Please, see my [PR](https://github.com/jnode-revisited/jnode-revisited.github.io/pull/8) or my fork [github/codehangen/jnode-revisited.github.io](https://github.com/codehangen/jnode-revisited.github.io)

#### concept: homepage
I would like to announce in this post the website: 'jnode-revisited' on GitHub Pages. Please, see this: [jnode-revisited.netlify.app](https://jnode-revisited.netlify.app)

![image](https://user-images.githubusercontent.com/111701513/208491876-681d4383-4dc1-46a1-9237-6f49e8fdd61c.png)

#### concept: documentation
![image](https://user-images.githubusercontent.com/111701513/208537854-d012345a-0635-4c2f-befd-e7b2b0fb8a8c.png)

#### question/feedback
*What do you all think of the idea?*

**Comment by tripleo1** [comment, marked as answer]
LGTM. +1

**Reply by ghost** [reply]
Hi tripleo1.

> LGTM. +1

thank you for feedback.
